custom_components/ramses_cc/services.yaml

```diff
@@ -133,33 +133,27 @@ set_system_mode:
             - custom
 
     period:
-      name: Days
+      name: Period (days)
       description: >-
-        Optional. A period of time in days; used only with away, day_off, or custom.
+        Optional. A period of time in days; used only with away, day_off, day_off_eco or custom.
         The system will revert to auto at midnight (up to 99 days, today is day 1).
-      default: 1
+      default:
+        days: 28
       example: '{"days": 28}'
       selector:
-        number:
-          min: 1
-          max: 99
-          step: 1
-          unit_of_measurement: days
-          mode: slider
+        duration:
+          enable_day: true
 
     duration:
-      name: Eco/Boost Duration
+      name: Duration (hours/minutes)
```

Comment:
Renamed to make it clearer what this field is when switching between the UI/YAML and referencing descriptions.